src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -272,6 +275,39 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
+                                        _ComputeReferenceAssemblies
+
+    Computes references that are only used at compile-time.
+    ============================================================
+    -->
+  <Target Name="_ComputeReferenceAssemblies">
+
+    <ItemGroup>
+      <_FrameworkReferenceAssemblies Include="@(ReferencePath)"
+                                     Condition="%(ReferencePath.FrameworkFile) == 'true' or
```

Comment:
Agreed.  The first place I see this line comes from:

```
Added Item(s): 
                       ResolvedFrameworkAssemblies=
                           System.Xml
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=System.Xml
                                   NuGetIsFrameworkReference=true
                                   NuGetPackageId=System.Xml.XPath.XDocument
                                   NuGetPackageVersion=4.3.0
                                   NuGetSourceType=Package
                                   PackageName=System.Xml.XPath.XDocument
                                   PackageVersion=4.3.0
                                   ParentPackage=System.Xml.XPath.XDocument/4.3.0
                                   ParentTarget=.NETFramework,Version=v4.6.1
                                   Private=false
                           System.Xml.Linq
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=System.Xml.Linq
                                   NuGetIsFrameworkReference=true
                                   NuGetPackageId=System.Xml.XPath.XDocument
                                   NuGetPackageVersion=4.3.0
                                   NuGetSourceType=Package
                                   PackageName=System.Xml.XPath.XDocument
                                   PackageVersion=4.3.0
                                   ParentPackage=System.Xml.XPath.XDocument/4.3.0
                                   ParentTarget=.NETFramework,Version=v4.6.1
                                   Private=false
                           mscorlib
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=mscorlib
                                   NuGetIsFrameworkReference=true
                                   NuGetPackageId=System.Xml.XPath.XDocument
                                   NuGetPackageVersion=4.3.0
                                   NuGetSourceType=Package
                                   PackageName=System.Xml.XPath.XDocument
                                   PackageVersion=4.3.0
                                   ParentPackage=System.Xml.XPath.XDocument/4.3.0
                                   ParentTarget=.NETFramework,Version=v4.6.1
                                   Private=false
```

From this target:
`Target "_ComputeLockFileFrameworks: (TargetId:16)" in file "F:\sdk\bin\Debug\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets" from project "F:\git\Mvc\test\WebSites\BasicWebSite\BasicWebSite.csproj" (target "ResolveLockFileReferences" depends on it):`

https://github.com/dotnet/sdk/blob/67fd3a133eb052fe396377b09b291cf32cd88212/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets#L346-L361

Looking to the input of that task, I see:

```
                            Microsoft.Win32.Registry/4.4.0-preview2-25328-01/mscorlib
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=mscorlib
                                   PackageName=Microsoft.Win32.Registry
                                   PackageVersion=4.4.0-preview2-25328-01
                                   ParentPackage=Microsoft.Win32.Registry/4.4.0-preview2-25328-01
                                   ParentTarget=.NETFramework,Version=v4.6.1
                           System.AppContext/4.3.0/mscorlib
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=mscorlib
                                   PackageName=System.AppContext
                                   PackageVersion=4.3.0
                                   ParentPackage=System.AppContext/4.3.0
                                   ParentTarget=.NETFramework,Version=v4.6.1
...
                           System.Xml.XPath.XDocument/4.3.0/mscorlib
                                   FileGroup=FrameworkAssembly
                                   FrameworkAssembly=mscorlib
                                   PackageName=System.Xml.XPath.XDocument
                                   PackageVersion=4.3.0
                                   ParentPackage=System.Xml.XPath.XDocument/4.3.0
                                   ParentTarget=.NETFramework,Version=v4.6.1
```

Looks like Xml.Xpath.XDocument is just the last one defined, so it "wins" at the end.